code/Modules/Input/emsc/emscInputMgr.h

```diff
@@ -58,9 +62,14 @@ class emscInputMgr : public inputMgrBase {
     static EM_BOOL emscDeviceMotion(int eventType, const EmscriptenDeviceMotionEvent* e, void* userData);
     /// device orientation callback
     static EM_BOOL emscDeviceOrientation(int eventType, const EmscriptenDeviceOrientationEvent* e, void* userData);
+    /// gamepad callback when connected
+    static EM_BOOL emscGamepadConnected(int eventType, const EmscriptenGamepadEvent* e, void* userData);
+    /// gamepad callback when disconnected
+    static EM_BOOL emscGamepadDisconnected(int eventType, const EmscriptenGamepadEvent* e, void* userData);
 
     static const int MaxNumKeys = 256;
     RunLoop::Id runLoopId;
+    RunLoop::Id updateGamepadsRunLoopId;
```

Comment:
the new member 'updateGamepadsRunLoopId' is not initialized in the constructor (if you want, you can also remove the constructor, and initialize all members inline in the class declaration, in new code I usually do it like this).